Base/QTApp/qSlicerMainWindow.h

```diff
@@ -64,6 +64,18 @@ class Q_SLICER_BASE_QTAPP_EXPORT qSlicerMainWindow : public QMainWindow
   /// \sa moduleSelector(), pythonConsole()
   Q_INVOKABLE ctkErrorLogWidget* errorLogWidget()const;
 
+  /// Read windows settings and update user interface.
+  ///
+  /// \sa writeSettings()
+  /// \sa qSlicerMainWindowPrivate::setupRecentlyLoadedMenu
+  Q_INVOKABLE void readSettings();
```

Comment:
Make them virtual so that more things can be saved in derived classes in custom applications.